set.mm

```diff
@@ -509765,111 +510466,380 @@ elements in a zero ring (see ~ 0ringirng ).  (Contributed by Thierry
       NZDYAXRFUVLTYOYPUUHYNGWNZWAWOUUMYBWPSUUMXQYHXBVGABMUUTUUPWQZPUUQMUUTWRABU
       UCUVNKACDXKXNFMYTGUUDHIWSWEMBUUTUUPWTXAXCXDYCYEVNYJYIYDYAYFXTYBUUHYBTUULX
       EXFXGABYSYAXQCDXSYFEXKXNLYTYPUUAHIUUEUUDUUFUULUUKJXHUVMXI $.
+
+    $d .0. q $.  $d A q $.  $d E q $.  $d F q $.  $d O q $.  $d P q $.
+    $d ph q $.
+    minplym1p.1 $e |- U = ( Monic1p ` ( E |`s F ) ) $.
+    $( A minimal polynomial is monic.  (Contributed by Thierry Arnoux,
+       2-Apr-2025.) $)
+    minplym1p $p |- ( ph -> ( M ` A ) e. U ) $=
+      ( vq cfv co eqid wcel syl csn ces1 c0g wceq cdm crab cress cig1p cbs cpl1
+      cv crsp cirng fldcrngd csdrg csubrg sdrgsubrg irngssv minplyval cdg1 cdif
+      sseldd cima cr clt cinf cdr clidl wne w3a sdrgdrng ply1annidl ply1annig1p
+      sneqd fveq2d eqtr4d crg drngringd ply1ring minplycl irngnminplynz neeqtrd
+      ressply10g pidlnz syl3anc eqnetrrd ig1pval3 simp2d eqeltrd ) ABFOZBNUJDEU
+      APZOODUBOZUCNWJUDUEZDEUFPZUGOZOZCABDUHOZWMUIOZWLDEWNWQUKOZFWJWKNWJQZWQQZW
+      PQZIJADEULPWPBAWPDEWMWJWKWSWMQZXAWKQZADIUMZAEDUNORZEDUOORJEDUPSZUQLVAZXCW
+      LQZWRQZWNQZKURZAWOWLRZWOCRZWOWMUSOZOXNWLWQUBOZTZUTVBVCVDVEUCZAWMVFRZWLWQV
+      GOZRWLXPVHXLXMXQVIAXEXRJEDWMXBVJSZABWPWQWLDEWJWKNWSWTXAXDXFXGXCXHVKAWITZW
+      ROZWLXPAYBWOTZWROWLAYAYCWRAWIWOXKVMVNABWPWQWLDEWNWRWJWKNWSWTXAIJXGXCXHXIX
+      JVLVOAWQVPRZWIWQUHOZRWIXOVHYBXPVHAWMVPRYDAWMXTVQWQWMWTVRSABWPWQWLDEWNWRFW
+      JWKNWSWTXAIJXGXCXHXIXJKVSAWIGXOABDEFGHIJKLVTAYEDDUIOZEWQWMGYFQXBWTYEQZXFH
+      WBWAYEWQWRWIXOYGXOQZXIWCWDWEXNWQWMXSWNWLCXOWTXJYHXSQXNQMWFWDWGWH $.
   $}
 
   ${
-    $d A p q $.  $d A q r $.  $d B p $.  $d E p q $.  $d E r $.  $d F p x $.
-    $d F q $.  $d G p q $.  $d G x $.  $d J p q $.  $d J x $.  $d K p q $.
-    $d L p q $.  $d L r $.  $d L x $.  $d N x $.  $d O p q $.  $d O r $.
-    $d P p q $.  $d P r $.  $d P x $.  $d Q p q $.  $d Q x $.  $d U p $.
-    $d U x $.  $d Z p $.  $d Z x $.  $d p ph q $.  $d ph r $.  $d ph x $.
     algextdeg.k $e |- K = ( E |`s F ) $.
     algextdeg.l $e |- L = ( E |`s ( E fldGen ( F u. { A } ) ) ) $.
+    algextdeg.d $e |- D = ( deg1 ` E ) $.
+    algextdeg.m $e |- M = ( E minPoly F ) $.
     algextdeg.f $e |- ( ph -> E e. Field ) $.
     algextdeg.e $e |- ( ph -> F e. ( SubDRing ` E ) ) $.
     algextdeg.a $e |- ( ph -> A e. ( E IntgRing F ) ) $.
+    $( Lemma for ~ algextdeg .  (Contributed by Thierry Arnoux, 2-Apr-2025.) $)
```

Comment:
Maybe it would be helpful to understand the proof for ~ algextdeg if the meaning of this (and the following) lemma(s) is described in more details: What are `L`, `F` and `K` in this context?

Reply:
Noted! I will propose something in the next PR.

Reply:
Fixed in #4788.